CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changelog
 =========
 
+- Updates `desc.examples`:
+    * `NCSX` now has a fixed current profile. Previously it used a fixed iota based on a
+    fit, but this was somewhat inaccurate.
+    * `QAS` has been removed as it is now redundant with `NCSX`.
+    * `ARIES-CS` has been scaled to the correct size and field strength.
+    * `WISTELL-A` is now a true vacuum solution, previously it approximated the vacuum
+    solution with fixed rotational transform.
+    * Flips sign of iota for `W7-X` and `ATF` to account for positive jacobian.
```

Comment:
why do we do this?

Reply:
previously I had flipped the sign of the boundary modes to give a positive jacobian but forgot to flip iota to match, this just fixes that.